src/app/modules/header/header-account-menu/header-account-menu.component.ts

```diff
@@ -1,37 +1,52 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, SimpleChanges } from '@angular/core';
 import { ConfigService } from '../../../services/config.service';
 import { User } from '../../../domain/user';
 import { HttpClient } from '@angular/common/http';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatDividerModule } from '@angular/material/divider';
+import { RouterModule } from '@angular/router';
+import { CommonModule } from '@angular/common';
+import { FlexLayoutModule } from '@angular/flex-layout';
 
 @Component({
   selector: 'app-header-account-menu',
   templateUrl: './header-account-menu.component.html',
-  styleUrls: ['./header-account-menu.component.scss']
+  styleUrl: './header-account-menu.component.scss',
+  standalone: true,
+  imports: [
+    CommonModule,
+    FlexLayoutModule,
+    MatButtonModule,
+    MatIconModule,
+    MatMenuModule,
+    MatDividerModule,
+    RouterModule
+  ]
 })
 export class HeaderAccountMenuComponent implements OnInit {
-  @Input()
-  user: User;
-
-  firstName: string = '';
-  lastName: string = '';
-  roles: string[] = [];
-  isPreviousAdmin: boolean = false;
-  logOutURL: string;
-  switchToOriginalUserURL = '/api/logout/impersonate';
+  protected firstName: string = '';
+  protected isPreviousAdmin: boolean = false;
```

Comment:
No need to set to false?
```suggestion
  protected isPreviousAdmin: boolean;
```